pkg/skaffold/kubernetes/context/context.go

```diff
@@ -30,26 +31,44 @@ var (
 )
 
 var (
-	currentConfigOnce sync.Once
-	currentConfig     clientcmdapi.Config
-	currentConfigErr  error
+	kubeConfigOnce sync.Once
+	kubeConfig     clientcmd.ClientConfig
+	kubeContext    string
 )
 
-// ResetCurrentConfig is used by tests
-func ResetCurrentConfig() {
-	currentConfigOnce = sync.Once{}
+// ResetConfig is used by tests
+func ResetConfig() {
```

Comment:
does this need to be public/exported? can we make it private?

Reply:
Yes, it can be private. Don't know why it was public so far.